run/helm/snap/README.md

```diff
@@ -0,0 +1,246 @@
+
+```
+$ cat <<EOF >> new_values.yaml
+autoload:
+  - https://github.com/intelsdi-x/snap-plugin-collector-cpu/releases/download/6/snap-plugin-collector-cpu_linux_x86_64
+  - https://github.com/intelsdi-x/snap-plugin-publisher-file/releases/download/2/snap-plugin-publisher-file_linux_x86_64
+  - https://raw.githubusercontent.com/intelsdi-x/snap-integration-kubernetes/master/examples/tasks/cpu-file.json
+EOF
+```
+
+With the command above we override default values adding list of URLs of two plugin binaries `snap-plugin-collector-cpu_linux_x86_64` and `snap-plugin-publisher-file_linux_x86_64`.
+The third URL downloads exemplary task for CPU plugin.
+
+Now running command:
+
+`$ helm install -f new_values.yaml stable/snap`
+
+will create your very own Snap release!
+
+Now you can list all the pods in order to verify if Snap pods are running correctly:
+
+`$ kubectl get pods --all-namespaces`
+
+The output should contain Snap pods:
+
+```
+NAMESPACE     NAME                                    READY     STATUS    RESTARTS   AGE
+default       snap-1kc54                              1/1       Running   0          1m
+default       snap-mshwn                              1/1       Running   0          1m
+default       snap-q0mkc                              1/1       Running   0          1m
+default       snap-q51xc                              1/1       Running   0          1m
+default       snap-rqk45                              1/1       Running   0          1m
+default       snap-wcklm                              1/1       Running   0          1m
+
+```
+
+Now running the command:
+
+`
+$ kubectl exec -ti <snap_pod> -- snaptel plugin list
+`
+
+will print information about two loaded plugins. And command:
+
+`$ kubectl exec -ti <snap_pod> -- snaptel task list`
+
+will provide information whether the metrics collection was successful or not. Use task watch to verify that tasks have been created correctly.
+`$ kubectl exec -ti <snap_pod> -- snaptel task watch <task_id>`
+
+To get more information on how to configure Snap plugins, please go to [Deep Dive](#8-deep-dive) section below.
+
+## 8. Deep dive
+
+There are two aspects to consider when running Snap in Kubernetes:
+
+* [Configuration of Snap Plugins](#configuration-of-snap-plugins)
+* [Preparing Snap Task](#preparing-snap-task)
+
+### Configuration of Snap Plugins
+
+Some of the plugins require more advanced configuration then those presented in the section above. Good example of such plugin is [Docker collector](https://github.com/intelsdi-x/snap-plugin-collector-docker).  The Docker collector allows to collect runtime metrics from Docker containers and its host machine. It gathers information about resource usage and performance characteristics. In order to configure this plugin we need to follow steps described below.
```

Comment:
The intention of this effort was to provision Snap chart that would allow user to choose their own plugins and run tasks. However we can of course discuss creating the solution to monitor kubernetes.